libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -267,7 +267,8 @@ public int hashCode() {
     }
 
     checkArgument(clockRate > 0);
-    return new RtpPayloadFormat(formatBuilder.build(), rtpPayloadType, clockRate, fmtpParameters);
+    return new RtpPayloadFormat(
+        formatBuilder.build(), rtpPayloadType, clockRate, fmtpParameters, mediaEncoding);
```

Comment:
should it also include the logic to accommodate RFC3016 Section 4.2:

> Unless specified by an out-of-band means, the resolution of the timestamp is set to its default value of 90 kHz.

Reply:
>    If the clock rate has been set to its default value and it is
   necessary to obtain the audio sampling rate, this can be done by
   parsing the "config" parameter (see the following example).

      m=audio 49230 RTP/AVP 96
      a=rtpmap:96 MP4A-LATM/90000
      a=fmtp:96 object=8; cpresent=0; config=9128B1071070